lib/sinon.js

```diff
@@ -7,7 +7,21 @@
  * Copyright (c) 2010-2013 Christian Johansen
  */
 "use strict";
-module.exports = require("./sinon/util/core");
+
+var match = require("./sinon/match");
+
+module.exports = exports = require("./sinon/util/core");
+
+exports.extend = require("./sinon/extend");
+exports.match = match;
+exports.typeOf = require("./sinon/typeOf");
+
+/*
+ * allow deepEqual to check equality of matchers through
+ * dependency injectection. Otherwise we get a circular
+ * dependency
+ */
+exports.deepEqual = exports.deepEqual.use(match);
```

Comment:
This is still changing the `core` "global". Once this is include, the only way to access `deepEqual` *without* matchers is to include it directly. This is a temporary problem. This will be fixed in the future when the commonjs-ification is completed and/or when `core` goes away.